core/bindings.rs

```diff
@@ -397,8 +398,9 @@ fn send(
 
   let control = match v8::Local::<v8::ArrayBufferView>::try_from(args.get(1)) {
     Ok(view) => {
-      let mut backing_store = view.buffer().unwrap().get_backing_store();
-      let backing_store_ptr = backing_store.data() as *mut _ as *mut u8;
+      let backing_store = view.buffer().unwrap().get_backing_store();
+      let backing_store_ptr =
+        unsafe { &mut *backing_store.get() } as *mut _ as *mut u8;
```

Comment:
Remove the from_raw parts stuff below.
Backingstore derefs to slice already.

Reply:
done